app/index.ts

```diff
@@ -15,6 +15,7 @@ const SOLVE_BUTTON_ID: string = "solve-button";
 enum PuzzleType {
     DragonFjord,
     JarringWords,
+	Tetromino,
```

Comment:
style: we use spaces for indent.

Reply:
Thanks for catching this! Sorry -- my editor defaults to tabs, and I thought I'd caught most of these.